bsp/boards/scum/memory_map.h

```diff
@@ -0,0 +1,72 @@
+
+
+
+//RFCONTRLLER Registers
+#define RFCONTROLLER_REG__CONTROL       *(unsigned int*)(AHB_RF_BASE + 0x00)
+#define RFCONTROLLER_REG__STATUS        *(unsigned int*)(AHB_RF_BASE + 0x04)
+#define RFCONTROLLER_REG__TX_DATA_ADDR  *(char**)(AHB_RF_BASE + 0x08)
+#define RFCONTROLLER_REG__TX_PACK_LEN   *(unsigned int*)(AHB_RF_BASE + 0x0C)
+#define RFCONTROLLER_REG__INT           *(unsigned int*)(AHB_RF_BASE + 0x10)
+#define RFCONTROLLER_REG__INT_CONFIG    *(unsigned int*)(AHB_RF_BASE + 0x14)
+#define RFCONTROLLER_REG__INT_CLEAR     *(unsigned int*)(AHB_RF_BASE + 0x18)
+#define RFCONTROLLER_REG__ERROR         *(unsigned int*)(AHB_RF_BASE + 0x1C)
+#define RFCONTROLLER_REG__ERROR_CONFIG  *(unsigned int*)(AHB_RF_BASE + 0x20)
+#define RFCONTROLLER_REG__ERROR_CLEAR   *(unsigned int*)(AHB_RF_BASE + 0x24)
+
+//RFTIMER Registers
+#define RFTIMER_REG__CONTROL            *(unsigned int*)(AHB_RFTIMER_BASE + 0x00)
+#define RFTIMER_REG__COUNTER            *(unsigned int*)(AHB_RFTIMER_BASE + 0x04)
+#define RFTIMER_REG__MAX_COUNT          *(unsigned int*)(AHB_RFTIMER_BASE + 0x08)
+#define RFTIMER_REG__COMPARE0           *(unsigned int*)(AHB_RFTIMER_BASE + 0x10)
+#define RFTIMER_REG__COMPARE1           *(unsigned int*)(AHB_RFTIMER_BASE + 0x14)
+#define RFTIMER_REG__COMPARE2           *(unsigned int*)(AHB_RFTIMER_BASE + 0x18)
+#define RFTIMER_REG__COMPARE3           *(unsigned int*)(AHB_RFTIMER_BASE + 0x1C)
+#define RFTIMER_REG__COMPARE4           *(unsigned int*)(AHB_RFTIMER_BASE + 0x20)
+#define RFTIMER_REG__COMPARE5           *(unsigned int*)(AHB_RFTIMER_BASE + 0x24)
+#define RFTIMER_REG__COMPARE6           *(unsigned int*)(AHB_RFTIMER_BASE + 0x28)
+#define RFTIMER_REG__COMPARE7           *(unsigned int*)(AHB_RFTIMER_BASE + 0x2C)
+#define RFTIMER_REG__COMPARE0_CONTROL   *(unsigned int*)(AHB_RFTIMER_BASE + 0x30)
+#define RFTIMER_REG__COMPARE1_CONTROL   *(unsigned int*)(AHB_RFTIMER_BASE + 0x34)
+#define RFTIMER_REG__COMPARE2_CONTROL   *(unsigned int*)(AHB_RFTIMER_BASE + 0x38)
+#define RFTIMER_REG__COMPARE3_CONTROL   *(unsigned int*)(AHB_RFTIMER_BASE + 0x3C)
+#define RFTIMER_REG__COMPARE4_CONTROL   *(unsigned int*)(AHB_RFTIMER_BASE + 0x40)
+#define RFTIMER_REG__COMPARE5_CONTROL   *(unsigned int*)(AHB_RFTIMER_BASE + 0x44)
+#define RFTIMER_REG__COMPARE6_CONTROL   *(unsigned int*)(AHB_RFTIMER_BASE + 0x48)
+#define RFTIMER_REG__COMPARE7_CONTROL   *(unsigned int*)(AHB_RFTIMER_BASE + 0x4C)
+#define RFTIMER_REG__CAPTURE0           *(unsigned int*)(AHB_RFTIMER_BASE + 0x50)
+#define RFTIMER_REG__CAPTURE1           *(unsigned int*)(AHB_RFTIMER_BASE + 0x54)
+#define RFTIMER_REG__CAPTURE2           *(unsigned int*)(AHB_RFTIMER_BASE + 0x58)
+#define RFTIMER_REG__CAPTURE3           *(unsigned int*)(AHB_RFTIMER_BASE + 0x5C)
+#define RFTIMER_REG__CAPTURE0_CONTROL   *(unsigned int*)(AHB_RFTIMER_BASE + 0x60)
+#define RFTIMER_REG__CAPTURE1_CONTROL   *(unsigned int*)(AHB_RFTIMER_BASE + 0x64)
+#define RFTIMER_REG__CAPTURE2_CONTROL   *(unsigned int*)(AHB_RFTIMER_BASE + 0x68)
```

Comment:
Hmm. I'm a little bit puzzled by this. In an ideal world, the HW team would generate this one file automatically, running a little tool on their HW design. So we want all the register-related `#define`s to be in  the same file. Is it possible to copy-paste things back to this file?